apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -163,7 +163,7 @@ const PythonlabView: React.FunctionComponent<
   };
 
   return (
-    <div className={moduleStyles.pythonlab}>
+    <div data-theme="Dark" className={moduleStyles.pythonlab}>
```

Comment:
We should either also apply this to `WebLab2View` or put the theme on the `Codebridge` component.

Reply:
You can look at web lab 2 at `s/allthethings/lessons/51/levels/1`

Reply:
Okay I was curious about this- do we for sure want dark mode in all of weblab 2?

Reply:
Troubleshooting a weird bug in weblab where it makes this weird view appear on the page... 
<img width="1270" alt="Screenshot 2025-04-04 at 11 59 59 AM" src="https://github.com/user-attachments/assets/0ab69150-b903-493b-99af-76714ea0fde0" />

Reply:
Okay weird weird. This is resolved when I add it to weblab2view instead, so I went with that approach instead of adding to Codebridge.

Reply:
weird...glad it was an easy fix